.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+- node
+
+deploy:
+  provider: script
+  script: npm start
+  on:
+    branch: master
```

Comment:
this won’t work. What this does is the following:

each time a build on the master branch passes, `npm start` is run on travis. But this command does not deploy anything, we have to replace it with an actual command to deploy something. For now, suggest leave out the entire section, lines 1 - 3 or good enough for now :)

Reply:
we can think about deployments later. One discussion will be to deploy to now, and if we want another one to deploy to glitch